### PR TITLE
Language Changes & Lore Pt. 1

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
- Renames Neo-Russkiya to Zvezhan
- Reworks the syllable structure of Zvezhan and Sol Common.
- Updates the descriptions of Sol Common, Zvezhan, and Galactic Common

## Why It's Good For The Game
- Neo-Russkiya is one of the last holdouts of the long-dead Space Russia meme. The USSP is becoming a lot more nuanced soon, so its language will also be so.
- Sol Common didn't have a very interesting gimmick to it, while this rework makes it more interesting as a language and story device.
- GalCom needed a better description.

## Testing

- Entered game
- Observed language descriptions
- Set telecomms to changed languages
- Languages work as advertised.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
 
  <hr>

## Changelog

:cl:
tweak: Altered names and descriptions of Zvezhan, GalCom, and SolCom
/:cl:

### CODE REVIEW

**Review by DGamerL — CHANGES_REQUESTED** [review]
Please rebuild TGUI and merge master

**Comment by AyIong** [comment]
> Please rebuild TGUI and merge master

Merge master and rebuild TGUI* 🤓

**Comment by Somniworld** [comment]
This PR has the Lore Team Seal of Quality (TM) 

Capture d'écran 2024-11-01 183122

Lore Team Seal

**Comment by dj-34** [comment]
As a native Russian speaker, I can say that it has absolutely nothing related to my language, perhaps even to the rest of the Slavic languages.
The syllables seem to me to be purely Chinese, and with the "naming" of the language I can't find any connection at all, neither with Chinese nor with Slavic languages, unfortunately.

**Comment by PopeDaveThe3th** [comment]
> As a native Russian speaker, I can say that it has absolutely nothing related to my language, perhaps even to the rest of the Slavic languages. The syllables seem to me to be purely Chinese, and with the "naming" of the language I can't find any connection at all, neither with Chinese (i tried my best to find something related in Chinese language) nor with Slavic languages, unfortunately.

The name doesn't really have any basis in any existing words.

In regards to the language itself, it was a lot harder to find syllables for Russian than it was for Chinese and Vietnamese that I used for the rest of the syllables, so if you'd like to provide some extra Russian ones I'd be glad to slot them in.

**Comment by PopeDaveThe3th** [comment]
In response to earlier critique, new syllables from Russian, Romanian, Polish, Thai, and Ukrainian have been added to Zvezhan to further flesh it out.

File: code/modules/mob/language.dm [hunk 1]
```diff
@@ -471,25 +472,22 @@
 	key = "0"
 	syllables = list ("honk","squeak","bonk","toot","narf","zub","wee","wub","norf")
 
-/datum/language/com_srus
-	name = "Neo-Russkiya"
-	desc = "Neo-Russkiya, a bastard mix of Gutter, Sol Common, and old Russian. The official language of the USSP. It has started to see use outside of the fringe in hobby circles and protest groups. The linguistic spirit of Sol-Gov criticisms."
+/datum/language/com_zvezhan
+	name = "Zvezhan"
+	desc = "The language of revolution across the stars, intially formed as a hybrid of Russian and Mandarin Chinese by early Cygni settlers before incorporating additional influences from across Eurasia. "
```
Comment (lewcc) [inline code/modules/mob/language.dm:477]:
```suggestion
	desc = "The language of revolution across the stars, initially formed as a hybrid of Russian and Mandarin Chinese by early Cygni settlers before incorporating additional influences from across Eurasia."
```

**Pull request merged**